src/utils.ts

```diff
@@ -52,5 +52,5 @@ export let get_range = (ps: IPositionedString): Range => {
 };
 
 export let getGoLangImportsCmd = (): string => {
-  return `${config.golang_executable} list -mod=readonly -f '{{ join .Imports "\\n" }}' ./...`;
+  return `${config.golang_executable} list -mod=readonly -f "{{.Imports}}" ./...`;
```

Comment:
probably issue might be related to the new line format used by windows., in windows newline is represented using \r\n, on the other hand *nix uses \n.

Reply:
But we don't need to complicate things with `join template function` as done in PR we don't complicated our processing by not using this template function. Do you think we need to add OS newline handling?

Reply:
It is up you, you can make use of whatever the logic which looks optimal for this usecase.

Just wanted to under whether this issue is related to newline ending

Reply:
I will approve once this has been tried out.

Reply:
@arajkumar attached screen shot from windows machine on various combination of join template function options tried.